Makefile

```
@@ -4,20 +4,19 @@ MANDIR = $(DESTDIR)$(PREFIX)/share/man/man1
 DOCDIR = $(DESTDIR)$(PREFIX)/share/doc/bcal
 STRIP ?= strip
 
-CFLAGS ?= -O3
 CFLAGS += -Wall -Wextra -Wno-unused-parameter -Werror
```

Please retain `-O3`. I understand there may be issues with the packaging guidelines... in that case please override it there instead of doing it here. I prefer to retain the `-O3`.

---

Would something like this be OK?

```suggestion
CFLAGS_OPTIMIZATION ?= -O3

CFLAGS += $(CFLAGS_OPTIMIZATION) -Wall -Wextra -Wno-unused-parameter -Werror
```

Then the optimization flags can be overridden without having to specify (and keep in sync) all the rest. Perhaps same with warning flags.

---

Awesome!